packages/web-components/src/components/cbp-form-field/cbp-form-field.stories.tsx

```diff
@@ -35,6 +35,79 @@ export default {
   },
 };
 
+
+
+function generateCheckboxes(checkboxes) {
+  const html = checkboxes.map(({ label, name, value, checked, disabled }) => {
```

Comment:
context isn't getting passed to the checkboxes so looks off when using context control on sample

Reply:
fixed